src/lib/rest/rest.cpp

```diff
@@ -1298,7 +1298,7 @@ static int connectionTreat
     //
     // Transaction starts here
     //
-    lmTransactionStart("from", ip, port, url);  // Incoming REST request starts
+    lmTransactionStart("from", "", ip, port, url);  // Incoming REST request starts
```

Comment:
We should know ... connectionTreat has read part of the message ... MHD should be able to tell.
I don't know, not very important.
If you want, create an issue about it. If not ... then don't :-)

NTC